test/evergreen.yml

```diff
@@ -2589,6 +2603,20 @@ tasks:
             ${test_env_vars|} ../../../tools/run_parallel.sh 'nice ../../../test/checkpoint/recovery-test.sh "${data_validation_stress_test_args}" WT_TEST.$t' 120
 
   - name: data-validation-stress-test-checkpoint-fp-hs-insert-s1
+    depends_on:
```

Comment:
The stress test options 3 and 5 are also applicable to non-timestamped tables. Can we include them also as part of this?

Reply:
thanks - i've included them in a715739. there's a successful patch build for these new commits in the ticket comments.